src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceConnection.java

```diff
@@ -15,5 +16,7 @@
      */
     public SalesForceConnection() {
         this.webDriver = DriverManager.getInstance().getNavigator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)